src/transformers/trainer_tf.py

```diff
@@ -105,6 +106,13 @@ def __init__(
         self.epoch_logging = 0
         self.eval_loss = tf.keras.metrics.Sum()
 
+        warnings.warn(
+            "The class `TFTrainer` is deprecated and will be removed in version 5 of Transformers. "
+            "We recommend using native Keras instead, by calling methods like `fit()` and `predict()` "
+            "directly on the model object. Detailed examples of the Keras style can be found in our "
+            "examples at https://github.com/huggingface/transformers/tree/master/examples/tensorflow", FutureWarning
```

Comment:
```suggestion
            "examples at https://github.com/huggingface/transformers/tree/master/examples/tensorflow",
            FutureWarning
```

This or running make style on your branch to make the formatting test happy.

Reply:
Already done! I spotted the issue as soon as the CI failed